packages/react-native-reanimated/src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -621,8 +651,11 @@ export function createAnimatedComponent(
         default: { collapsable: false },
       });
 
+      const skipEntering = this.context?.current;
+
       return (
         <Component
+          nativeID={skipEntering ? undefined : `${this.nativeID}`}
```

Comment:
Can't we set it in the `nativeID` member and just pass here `nativeID={nativeID}`?